web-registry/src/components/organisms/CreditBatches/CreditBatches.tsx

```diff
@@ -126,16 +132,22 @@ const CreditBatches: React.FC<CreditBatchProps> = ({
         </Box>
       ))}
       onTableChange={onTableChange}
-      rows={batches.map(batch =>
+      rows={batches.map(batch => {
         /* eslint-disable react/jsx-key */
-        [
-          <Link
-            href={getHashUrl(batch.txhash)}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {truncateHash(batch.txhash)}
-          </Link>,
+        let result = [];
+        if (someTx) {
+          result.push(
+            <Link
+              href={getHashUrl(batch.txhash)}
+              target="_blank"
+              rel="noopener noreferrer"
```

Comment:
If this Link component is the one from our codebase then these attributes should be added automatically I think.